source/localizable/analytics/index.html.md.erb

```diff
@@ -26,56 +32,50 @@ section.
 > You **have to** replace the `'SA-XXXX-YYYY'` property with the actual
 *Shop Account ID* you received from us.
 
-## How it works
+### Step 2: Declare Ecommerce Data
+
+Send your *Order* and *Item* data with the <%= link_to 'ecommerce', 'ecommerce/' %> commands. Ecommerce
+commands **must be** added at the final step of your checkout process after a user has successfully completed a purchase.
+Usually this is the *"Thank You"* or *"Receipt"* page.
 
-> The source of the tracking script is hosted on
-[GitHub](https://www.github.com/skroutz/analytics.js).
+> ##### Note
+> You **must** first add the Tracking Code JS snippet, connect to your Account and then add the ecommerce commands.
 
-The tracking script, registers a new global function named `sa`, and then
-asynchronously loads the `analytics.js` library onto the page.
+#### Tracking an Order
 
-The `sa` global function is the main way you interact with the Analytics library.
-It accepts a sequence of parameters, that represent:
+Use the <%= link_to 'ecommerce:addOrder', 'ecommerce/#addorder' %> command to track the order.
 
-Category
-: The API category
+<%= render_code_from_file 'analytics/order_beacon' %>
 
-Command
-: The Command of the API category you want to be executed
+#### Tracking Items
 
-Data
-: The Data to be reported by the Command
+Use the <%= link_to 'ecommerce:addItem', 'ecommerce/#additem' %> command to track the items of the order.
 
-> ##### Note
-> *Categories* and *Commands* are **case-sensitive**.
+<%= render_code_from_file 'analytics/item_beacon' %>
 
-## Next Steps
+#### Complete Example
 
-* Configuration
-  * <%= link_to 'Analytics Settings', '/analytics/settings' %>
-  * <%= link_to 'Ecommerce metrics', '/analytics/ecommerce' %>
-* <%= link_to t('docs.analytics.examples'), '/analytics/examples' %>
+<% tracking_script = YAML.load_file(File.expand_path(File.join('./resources',
+                                                               'code_examples',
+                                                               "analytics/#{flavor}/js_snippet.yml"))) %>
+<% complete_example = YAML.load_file(File.expand_path(File.join('./resources',
```

Comment:
We already did that with the tracking script. We just extracted the `complete_example` as well.